Diffusion/Common/State.swift

```diff
@@ -24,8 +24,16 @@ enum GenerationState {
 
 typealias ComputeUnits = MLComputeUnits
 
+/// Schedulers compatible with StableDiffusionPipeline
+public enum Diffusion_StableDiffusionScheduler: String {
```

Comment:
Not very fond of this name, the complete type will be `Diffusion.Diffusion_StableDiffusionScheduler`.

I would suggest:
- Call it `Scheduler` (or even `StableDiffusionScheduler`, and then use `StableDiffusion.StableDiffusionScheduler` when we need to disambiguate).
- Use it like that everywhere.
- Write an extension `var asStableDiffusionScheduler: StableDiffusionScheduler`
- Use the extension when assigning to the configuratin.

Such an extension could look like this, assuming we call the type `Scheduler`:

```Swift
extension Scheduler {
    var asStableDiffusionScheduler: StableDiffusionScheduler {
        switch self {
        case .pndmScheduler: return StableDiffusionScheduler.pndmScheduler
        case .dpmSolverMultistepScheduler: return StableDiffusionScheduler.dpmSolverMultistepScheduler
        }
    }
}
```